carbon-location.html

```diff
@@ -148,11 +148,18 @@
         }
       },
 
+      ready: function() {
+        this._isReady = true;
```

Comment:
There is, and that's what Steve used when we were testing this, but I wasn't sure if it was kosher to  use that, as it's private to Polymer.

I did a quick search and I couldn't find any uses of it anywhere, in our elements or in other folks' code that I could easily search.